WTWSMS/history/characters/greek.txt

```diff
@@ -3802,6 +3827,18 @@
 		death=yes
 	}
 }
+13100018 = {
+    name="Callinicus" #PLRE vol.3 Ioannes 1
```

Comment:
Looks like an error in the comment.

Reply:
I can confirm, I thought about changing it long ago but I never remember doing it.